src/components/10-atoms/checkbox/index.scss

```diff
@@ -128,4 +128,10 @@ axa-checkbox {
         }
       }
     }
-}
+    .a-checkbox__children-inline {
+      * {
```

Comment:
Did you need that for a specific element? Because this wildcard operator can have weird effects if somebody wants to add to whatever is within `.a-checkbox__children-inline`.

Reply:
I need it for every element a user passes through the component